beacon/engine/types.go

```diff
@@ -261,10 +260,12 @@ func BlockToExecutableData(block *types.Block, fees *big.Int, blobs []kzg4844.Bl
 		Blobs:       make([]hexutil.Bytes, 0),
 		Proofs:      make([]hexutil.Bytes, 0),
 	}
-	for i := range blobs {
-		blobsBundle.Blobs = append(blobsBundle.Blobs, hexutil.Bytes(blobs[i][:]))
-		blobsBundle.Commitments = append(blobsBundle.Commitments, hexutil.Bytes(commitments[i][:]))
-		blobsBundle.Proofs = append(blobsBundle.Proofs, hexutil.Bytes(proofs[i][:]))
+	for i := range sidecars {
+		for j := range sidecars[i].Blobs {
+			blobsBundle.Blobs = append(blobsBundle.Blobs, hexutil.Bytes(sidecars[i].Blobs[j][:]))
+			blobsBundle.Commitments = append(blobsBundle.Commitments, hexutil.Bytes(sidecars[i].Commitments[j][:]))
+			blobsBundle.Proofs = append(blobsBundle.Proofs, hexutil.Bytes(sidecars[i].Proofs[j][:]))
+		}
```

Comment:
```suggestion
	for _, sidecar := range sidecars {
		for j, blob := range sidecar.Blobs {
			bundle.Blobs = append(bundle.Blobs, hexutil.Bytes(sidecar.Blobs[j][:]))
			bundle.Commitments = append(bundle.Commitments, hexutil.Bytes(sidecar.Commitments[j][:]))
			bundle.Proofs = append(bundle.Proofs, hexutil.Bytes(sidecar.Proofs[j][:]))
		}
```
make it a _bit_ less dense, and remove the possibility of mixing up `i` and `j`